.helm/ecamp3/values.yaml

```diff
@@ -73,6 +73,18 @@ print:
     port: 3003
   replicaCount: 1
 
+browserless:
+  enabled: true
+  domain:
+  image:
+    repository: "docker.io/browserless/chrome"
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose shared default is .Values.imageTag
+    tag: "1.50.0-puppeteer-12.0.1"
```

Comment:
We should probably tell renovate about this tag.

Reply:
Any good idea how to achieve this? It's a bit tricky, because we have 2 versions included, the version of browserless and the version of puppeteer.